docarray/array/storage/elastic/seqlike.py

```diff
@@ -79,3 +79,4 @@ def extend(self, values: Iterable['Document']) -> None:
         if len(request) > 0:
             self._send_requests(request)
             self._refresh(self._config.index_name)
+            self._update_offset2ids_meta()
```

Comment:
I see for other backends `_update_offset2ids_meta` is called only in `_save_offset2ids` on delete, I guess you encountered an issue when docs are extended but not properly saved?